### PR TITLE
Disposes trace panel when trace removed

### LINKED ISSUE #104: Trace panel not closed when trace is removed from server
If a trace is removed from the `'OPENED TRACES` view using the remove button, then corresponding trace panel is not closed.

vscode-remove-bug

### PR DESCRIPTION
**Pull request by ngondalia**
* Appropriate signal handling for removed traces
* Edited behaviour for the "X" button in the "opened traces" view

Fixes #104

Signed-off-by: Neel Gondalia ngondalia@blackberry.com

### CODE REVIEW

File: vscode-trace-webviews/src/trace-explorer/opened-traces/vscode-trace-explorer-opened-traces-widget.tsx [hunk 1]
```diff
@@ -78,10 +89,14 @@ class TraceExplorerOpenedTraces extends React.Component<{}, OpenedTracesAppState
 
   componentDidMount(): void {
       this._signalHandler.notifyReady();
+      // ExperimentSelected handler is registered in the constrictor (upstream code), but it's
```
Comment (bhufmann) [inline vscode-trace-webviews/src/trace-explorer/opened-traces/vscode-trace-explorer-opened-traces-widget.tsx:92]: typo: `constrictor` -> `constructor`

**Review by bhufmann — CHANGES_REQUESTED** [review]
Works well... just a small typo to fix. Thanks!

**Review by bhufmann — CHANGES_REQUESTED** [review]
LGTM. Thanks!

**Review by bhufmann — APPROVED** [review]

**Comment by hoangphamEclipse** [comment]
LGTM.

**Pull request merged**